bitnami/elasticsearch/templates/coordinating-svc.yaml

```diff
@@ -20,5 +20,7 @@ spec:
       {{- else if eq .Values.coordinating.service.type "ClusterIP" }}
       nodePort: null
       {{- end }}
+    - name: tcp-transport
+      port: 9300
```

Comment:
Why is this necessary? This service is intended for "coordinating-only" nodes. Why do you need to access this port on this nodes?

Reply:
This Service is reached on port 9300 from the other nodes, which use this port for different reasons, for example for zen discovery and cluster assembling or when an HTTP call is made to one node and a different one is holding the data.
This is needed to monitor this kind of traffic

Reply:
Agree on this! We can expose 9300 on coordinating-only nodes

Reply:
The comment above applies to all nodes, since they use 9300 to assemble the cluster and also to periodically check on each others presence.